pkg/systemd/services/services.jsx

```diff
@@ -704,8 +704,8 @@ class ServicesPage extends React.Component {
                     if (unit.LoadState == "not-found")
                         return false;
 
-                    if (currentTextFilter && unit.Description && unit.Description.toLowerCase().indexOf(currentTextFilter) == -1 &&
-                        unit.Id.indexOf(currentTextFilter) == -1 && !(unit.Names || []).some(name => name.indexOf(currentTypeFilter) != -1))
+                    if (currentTextFilter && !((unit.Description && unit.Description.toLowerCase().indexOf(currentTextFilter.toLowerCase()) != -1) ||
+                        unit_id.indexOf(currentTextFilter) != -1))
```

Comment:
Shouldn't then we always check case insensitive even for `unit_id`?

Reply:
Done.